src/components/Search/SearchResultsListItem.tsx

```diff
@@ -110,24 +137,26 @@ function SearchResultsListItem({
                   {revisionHash}
                 </Typography>
 
-                <div className='info-caption'>
-                  <div className='info-caption-item item-author'>
-                    {' '}
-                    <MailOutlineOutlinedIcon
-                      className='mail-icon'
-                      fontSize='small'
-                    />{' '}
-                    {item.author}
-                  </div>
+                <Tooltip title={utcDateTime} placement='top'>
```

Comment:
I think that the tooltip should be on the date time only, if that's possible.

Reply:
I have tweaked this